05.ios/04.safari-web-extension/docs.en.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'AdGuard Safari Web Extension'
```

Comment:
"Web Extension"

In the text:

AdGuard's Safari [Web extension](https://developer.apple.com/documentation/safariservices/safari_web_extensions)